LICENSE

```diff
@@ -0,0 +1,15 @@
+ISC License
+
+Copyright (c) {{ year }}, {{ author }}
```

Comment:
Don't we need to fill this out?

Reply:
`2022, Power Home Remodeling` how should that last part be?